lib/beaker/host/unix/pkg.rb

```diff
@@ -93,14 +93,14 @@ def install_package(name, cmdline_args = '', version = nil, opts = {})
         end
         update_apt_if_needed
         execute("apt-get install --force-yes #{cmdline_args} -y #{name}", opts)
-      when /solaris-11/
-        if opts[:acceptable_exit_codes]
-          opts[:acceptable_exit_codes] << 4
-        else
-          opts[:acceptable_exit_codes] = [0, 4] unless opts[:accept_all_exit_codes]
+      when /solaris-/
+        if self['platform'] =~ /solaris-11/
+          if opts[:acceptable_exit_codes]
```

Comment:
@kevpl Do you recall the reasoning behind adding 4 to the acceptable exit codes for Solaris 11? I've preserved that code here, but I couldn't find any explanation for what it was there for in the previous commit. If it was due to some oddity with the pkg command, then I could probably remove this entirely. However, I did take a look at the man page for pkg and it makes no reference to a return code of 4, so I'm not sure.

Reply:
@ScottGarman yeah, looking back at [the commit](https://github.com/puppetlabs/beaker/commit/a661dfe0dab92991808d7d4e9cbdee66afcf7022), I'm not able to find an explanation for it either. The way the code is formatted though, it would seem that it's really only there for the `pkg` command, so I think you should delete it since we're no longer using that command at all